include/smack/SmackRep.h

```diff
@@ -150,6 +166,14 @@ class SmackRep {
   virtual const Stmt* alloca(llvm::AllocaInst& i);
   virtual const Stmt* memcpy(const llvm::MemCpyInst& msi);
   virtual const Stmt* memset(const llvm::MemSetInst& msi);
+  virtual const Stmt* loadAsBytes(const llvm::LoadInst& li);
+  virtual const Stmt* storeAsBytes(const llvm::StoreInst& si);
```

Comment:
Yes, you are right. In this way, the declaration of load/store methods would be,

`virtual const Stmt* load(const llvm::Value* addr, const llvm::Value* val, const llvm::Instruction * I);`
`virtual const Stmt* store(const llvm::Value* addr, const llvm::Value* val, const llvm::Instruction *I);` 

since we could not get the `llvm::Function` object from either `llvm::Value* addr` or `llvm::Value* val`. If we agree on this, then the declaration could stay the same for we can get `addr` and `val` from instruction.